src/uwtools/tests/test_schemas.py

```diff
@@ -594,6 +601,32 @@ def test_schema_make_hgrid_run_dir(make_hgrid_prop):
     assert "88 is not of type 'string'" in errors(88)
 
 
+# make_solo_mosaic
+
+
+def test_schema_make_solo_mosaic():
+    config = {
+        "config": {"dir": "path/to/dir", "num_tiles": 1},
+        "execution": {"executable": "make_solo_mosaic"},
+        "run_dir": "/tmp",
+    }
+    errors = schema_validator("make-solo-mosaic", "properties", "make_solo_mosaic")
+    # Basic correctness:
+    assert not errors(config)
+    # All top-level keys are required:
+    for key in ("config", "execution", "run_dir"):
+        assert f"'{key}' is a required property" in errors(with_del(config, key))
+    # Additional top-level keys are not allowed:
+    assert "Additional properties are not allowed" in errors({**config, "foo": "bar"})
+
+
```

Comment:
I think we probably want a `test_schema_make_solo_mosaic_config()` here to exercise the `config:` block -- verifying that validation fails a config lacking the required items, that it fails items with incorrectly typed values, etc. (maybe not etc., maybe that's it). Probably this is related to a question asked earlier about what should _be_ in the schema.